index.html

```diff
@@ -208,90 +210,101 @@ <h2 class="text-3xl font-extrabold text-gray-900 leading-9 sm:text-4xl sm:leadin
 
       return currency + price.toString()
     }
+
+    function calculateBubblePosition(volumeIndex) {
+      const range = document.getElementById("volume")
+      const newVal = Number((volumeIndex / range.max) * 100)
+      return `left: calc(${newVal}% + (${13.87 - newVal * 0.26}px))`
```

Comment:
The browser's default slider thumb size is 10px. Ours is 26px, so I had to inflate those constants here to align the `<output>` tag accordingly.